src/components/shared/ObjectInspector.js

```diff
@@ -122,6 +122,26 @@ const ObjectInspector = React.createClass({
   getChildren(item: ObjectInspectorItem) {
     const { getObjectProperties } = this.props;
     const { actors } = this;
+    const key = item.path;
```

Comment:
nit: this can probably move into this ` if (item.contents.value && item.contents.value.preview && actors[key]) {` line below, since is used only within it